### PR TITLE
Add zh-Hans translation

### PR DESCRIPTION
**Pull request by TC999**

### CODE REVIEW

**Review by kbinani — CHANGES_REQUESTED** [review]
Could you please revert the changes made to `resource/japanese.lang`? The correct character in Japanese for '转换' is '変換'. Thank you.

File: resource/chinese.lang [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+"or" = "或者"
+"Next" = "继续"
+"Select from other directories" = "选择其他文件夹"
+"Save" = "保存"
+"Cancel" = "取消"
+"Confirmation" = "确认"
+"Selected world:" = "已选择世界："
+"Start" = "开始"
+"Back" = "返回"
+"Converting..." = "変換中..."
```
Comment (kbinani) [inline resource/chinese.lang]: I believe this line was meant to be `"Converting..." = "转换中..."`. It seems to have been swapped with the entry in `resource/japanese.lang`.

**Comment by TC999** [comment]
OK

**Comment by kbinani** [comment]
The changes to `.github/workflows/main.yml` are unnecessary, so I will change the target branch of this PR to zh-hans-translation-202411 and merge it for now.

**Comment by kbinani** [comment]
This change was merged to main in https://github.com/kbinani/je2be-desktop/pull/10
@TC999 Thank you for your contribution

**Pull request merged**